ios/RCTPdf/RCTPdfView.m

```diff
@@ -118,6 +118,7 @@ - (void)didSetProps:(NSArray<NSString *> *)changedProps
 
         if ([changedProps containsObject:@"path"]) {
 
+            _path = (__bridge_transfer NSString *)CFURLCreateStringByReplacingPercentEscapes(NULL, (CFStringRef)_path, CFSTR(""));
```

Comment:
Shouldn't this be done at line 135? I don't think we should touch blob urls.

Reply:
added to line 135